examples/etw_threads/main.cc

```diff
@@ -113,7 +113,7 @@ void beep()
 // Max number of threads to spawn
 constexpr int kMaxThreads = 10;
 
-int main(int arc, char **argv)
+int main()
```

Comment:
Why ?

If the issue is that argc and argv are not used, just comment the parameter names, but do not change the prototype of main.

Reply:
Noted. Added the **commented parameter names**.